docs/source/user-guide/vision/supported_tasks_and_formats.rst

```diff
@@ -0,0 +1,124 @@
+.. _supported_tasks_and_formats:
+
+===========================
+Supported Tasks and Formats
+===========================
+
+In order to accurately analyze the images, labels and predictions provided in the
+:doc:`VisionData object </user-guide/vision/vision_data_class>`,
+they must be provided in deepchecks accepted format. In this section we will describe the supported formats
+for each supported task type.
+
+The data formats are validated at the creation of the VisionData object based on the first batch of data. However,
+in addition to the automatic validations, it is also important to make sure visually that the data was provided in the
+correct format via the :func:`head <deepchecks.vision.VisionData.head>` functionality.
```

Comment:
Also a good place to mention that data must be shuffled by the user.

Reply:
Weird place for it IMO